### DISCUSSION TITLE
Issue with Brew Installed MacOS Applications that were already installed using .dmg

### DISCUSSION THREAD

**Discussion by yazanzaid00** [Homebrew/discussions#5007] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.0
ORIGIN: https://github.com/Homebrew/brew
HEAD: ff899e5b3ff6731208564ef502db237e2627e8ba
Last commit: 5 days ago
Core tap JSON: 23 Dec 01:04 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.2.1-arm64
CLT: 15.1.0.0.1.1700200546
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

I've installed MacOS applications via Brew that were previously installed using their respective .dmg/.pkg files (from mounted disks). How can I clean up these redundant packages? Is resetting my MacOS the best solution to undo this action?

### Relevant casks

microsoft-edge, nvm, adguard, google-drive, Logi-options-plus, logi-g-hub, mathpix, photosync, raycast, rectangle-pro, vscode

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Tips-N'-Tricks#appoint-homebrew-cask-to-manage-a-manually-installed-app

**Reply by yazanzaid00** [reply]
Installed App with Brew Cask Over Existing Install from .dmg/.pkg file, Did it Remove the Old App Files?
